### DISCUSSION TITLE
GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE - getReports giving reports for multiple Market IDs

### DISCUSSION THREAD

**Discussion by yashraj021** [amzn/selling-partner-api-models#4077] [Q&A] [answered]
Hi,

I am trying to fetch settlement report using getReports API call where I am mentioning only one marketplaceId, i.e, USA, however I am getting result for multiple Market Place IDs. 

Ex-URL: `https://sellingpartnerapi-na.amazon.com/reports/2021-06-30/reports?reportTypes=GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE&marketplaceIds=ATVPDKIKX0DER`.

Attaching SS below:

<img width="1068" alt="Screenshot 2024-08-05 at 4 35 21 PM" src="https://github.com/user-attachments/assets/17258052-c9d4-475b-979e-450d55d0114b">

**Comment by mafge** [comment, marked as answer]
Hey @yashraj021,
thanks for reaching out!
Please be aware that the logic behind the marketplace id parameter is to return reports that match at least one of the specified marketplace ids (see also description for the parameter in [the docs](https://developer-docs.amazon.com/sp-api/docs/reports-api-v2021-06-30-reference#parameters)). Hope that helps you to understand the result set that you see for your reports!
Best,
Marc
Selling Partner Developer Services

**Reply by yashraj021** [reply]
> Hey @yashraj021, thanks for reaching out! Please be aware that the logic behind the marketplace id parameter is to return reports that match at least one of the specified marketplace ids (see also description for the parameter in [the docs](https://developer-docs.amazon.com/sp-api/docs/reports-api-v2021-06-30-reference#parameters)). Hope that helps you to understand the result set that you see for your reports! Best, Marc Selling Partner Developer Services

Then whats the point of having it?
